lib/chef/resource/windows_task.rb

```diff
@@ -116,6 +116,10 @@ class WindowsTask < Chef::Resource
                  introduced: "14.4", default: false,
                  description: "Scheduled task option when system is switching on battery."
 
+        property :description, String,
+                 introduced: "14.7",
+                 description: "Specifies the task description."
```

Comment:
@kagarmoe What would you like this field to say since it'll end up in the docs

Reply:
Leave off "specifies". Try: "The task description" or "Describes the task"